### PR TITLE
feat(route): add mof (bond management) 中华人民共和国财政部-专题-政府债券管理

### PR DESCRIPTION
**Pull request by la3rence**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/gov/mof/bond/
/gov/mof/bond/gzfxgzdt
/gov/mof/bond/gzfxzjs
/gov/mof/bond/gzfxdzs
/gov/mof/bond/difangzhengfuzhaiquan
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [X] New Route / 新的路由
  - [X] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [X] Documentation / 文档说明
- [X] Full text / 全文获取
  - [X] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [X] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [X] Parsed / 可以解析
  - [X] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

https://gks.mof.gov.cn/ztztz/guozaiguanli/gzfxgzdt/

### CODE REVIEW

File: lib/v2/gov/maintainer.js [hunk 1]
```diff
@@ -100,4 +100,5 @@ module.exports = {
     '/xinyi/:path+': ['ShuiHuo'],
     '/xuzhou/hrss/:category?': ['nczitzk'],
     '/zhejiang/gwy/:category?/:column?': ['nczitzk'],
+    '/mof/bond/:category?': ['la3rence'],
```
Comment (TonyRL) [inline lib/v2/gov/maintainer.js:103]: Please insert new entries in alphabetical order.

File: lib/v2/gov/router.js [hunk 2]
```diff
@@ -92,4 +92,5 @@ module.exports = function (router) {
     router.get(/xinyi(\/[\w/-]+)?/, require('./xinyi/xinyi'));
     router.get('/xuzhou/hrss/:category?', require('./xuzhou/hrss'));
     router.get('/zhejiang/gwy/:category?/:column?', require('./zhejiang/gwy'));
+    router.get('/mof/bond/:category?', require('./mof/bond'));
```
Comment (TonyRL) [inline lib/v2/gov/router.js:95]: Please insert new entries in alphabetical order.

**Review by TonyRL — COMMENTED** [review]
Missing corresponding `radar.js`

File: lib/v2/gov/maintainer.js (hunk 1 above)
Comment (la3rence) [reply lib/v2/gov/maintainer.js]: fixed.

File: lib/v2/gov/router.js (hunk 2 above)
Comment (la3rence) [reply lib/v2/gov/router.js]: fixed.

**Comment by la3rence** [comment]
> Missing corresponding `radar.js`

Is now a must thing? I don't use the plugin...
Anyway, added.

File: website/docs/routes/government.mdx [hunk 3]
```diff
@@ -1654,6 +1654,18 @@
 
 <Route author="HenryQW" example="/embassy/us/chicago" path="/embassy/:country/:city" paramsDesc={['国家短代码, 见支持国家列表', '城市, 对应国家列表下的`领事馆城市列表`']} />
 
+## 中华人民共和国财政部 {#zhong-hua-ren-min-gong-he-guo-cai-zheng-bu}
+
+### 专题 {#zhong-hua-ren-min-gong-he-guo-cai-zheng-bu-zhuan-ti}
+
+<Route author="la3rence" example="/gov/mof/bond/" path="/gov/mof/bond/:category?" paramsDesc={['专题，见下表，默认为国债管理工作动态']} radar="1">
```
Comment (TonyRL) [inline website/docs/routes/government.mdx:1661]:
```suggestion
<Route author="la3rence" example="/gov/mof/bond" path="/gov/mof/bond/:category?" paramsDesc={['专题，见下表，默认为国债管理工作动态']} radar="1">
```

**Pull request merged**